common/beerocks/bwl/dummy/ap_wlan_hal_dummy.cpp

```diff
@@ -173,6 +173,22 @@ bool ap_wlan_hal_dummy::sta_bss_steer(const std::string &mac, const std::string
     return true;
 }
 
+bool ap_wlan_hal_dummy::update_vap_credentials(
+    std::list<son::wireless_utils::sBssInfoConf> &bss_info_conf_list)
+{
+    for (auto bss_info_conf : bss_info_conf_list) {
```

Comment:
please use const reference
`const auto &bss_info_conf`

I guess you can use const on the function signature ass well.